onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -359,6 +359,19 @@ TEST(NnapiExecutionProviderTest, TestQDQMul) {
                   });
 }
 
+TEST(NnapiExecutionProviderTest, TestQDQTranspose) {
+  // NNAPI use different rounding, which may cause ~1% difference in the result
+  RunQDQModelTest(BuildQDQTransposeTestCase<uint8_t /* InputType */,
+                                            uint8_t /* OutputType */>(
+                      {1, 3, 32, 32} /* input_shape */,
+                      {0, 3, 1, 2} /* perms */),
+                  "nnapi_qdq_test_graph_transpose",
+                  {
+                      true /* verify_entire_graph_use_ep */,
+                      1e-2f /* fp32_abs_err */,
+                  });
```

Comment:
Do you need this here? Transpose is just moving data around without calculation.